### ISSUE TITLE
keyboard focus is not visible on the settings dropdown and checkboxes present in the setting dropdown.

### ISSUE THREAD

**Issue by msftedad** [docsifyjs/docsify#2410]
#### Steps to reproduce

1. Open [TypeDoc - v0.25.13](https://typedoc.org/api/) in edge browser.
2. Navigate to settings dropdown and observe the keyboard on it, and after invoking it observe the keyboard focus on the checkboxes.

#### Current behaviour

While navigate to settings dropdown, keyboard focus is not visible on the 'settings' dropdown and on the checkboxes present in the setting dropdown.

https://github.com/docsifyjs/docsify/assets/93735775/371e9055-895e-47bc-a36f-8cfeacdc2d40

#### Expected behaviour

keyboard focus should be visible on the 'settings' dropdown and on the checkboxes present in the setting dropdown.

#### Other relevant information

- Docsify version: 0.25.13

- [ ] Bug still occurs when all/other plugins are disabled?

- Docsify plugins (if the bug happens when plugins enabled, please try to isolate the issue):

#### Please create a reproducible sandbox

[TypeDoc - v0.25.13](https://typedoc.org/api/)

**Comment by paulhibbitts** [comment]
This issue is related to TypeDoc (https://github.com/TypeStrong/typedoc/tree/master) and not Docsify.
